packages/snap-preact-components/src/components/Atoms/Button/Button.stories.tsx

```diff
@@ -66,6 +67,18 @@ export default {
 			},
 			control: { type: 'color' },
 		},
+		icon: {
+			description: 'Button Icon name',
```

Comment:
Yea, we might want this in Storybook eh?  ;)

Nice catch.

Reply:
I think this came from Chris so props to him for this catch! https://github.com/searchspring/snap/commit/92bdb99c25955b17995671d6475697534152ee1d#diff-af43f9ca968d24cd962c27d74c9fa94a839d9094bbe15052a237d341fd9754c3R71